README.md

```diff
@@ -245,6 +245,7 @@ To run the tests behind proxy using Notebook:
    * katib
    * kfp_v2
    * kserve
+   * mlflow
    * mlflow-kserve
    * mlflow-minio
    * training
```

Comment:
@NohaIhab  From my understanding, `kfp_v1` is still not supported. But since it's for an old version of KFP, do you think it makes sense to just delete this note altogether? Or maybe include a smaller note saying that all UATs except for `kfp_v1` are supported?

Reply:
If someone tries to run the UATs, the `kfp_v1` will be included, it's even still included in our CI runs for the UATs.
We need to consider if a task is needed to configure `kfp_v1` behind proxy too, or if we want to deprecate them. Can you add it as a daily note? @mvlassis